### ISSUE TITLE
Allow for better control on when this extension suggests a method name

### ISSUE THREAD

**Issue by RobThree** [priyanshu92/AsyncMethodNameFixer#8]
It suggests MainAsync...

````c#
private static async Task Main(string[] args) {
}
````
Also: I'd like for it to be able to ignore (for example) controllers:

````c#
public class CustomerController : ControllerBase {
    public IAsyncResult<Customer> Get(int id) {
    }
}
````

...and testmethods from unittests:

````c#
[TestMethod]
public Task Foo() {
}
````

**Comment by RobThree** [comment]
By the way, this can be worked around with some compiler directives:

```c#
#pragma warning disable AMNF0001
    private static async Task Main(string[] args) {
#pragma warning restore AMNF0001
        // ....
    }
````
...but it's not exactly pretty...

**Comment by priyanshu92** [comment]
> It suggests MainAsync...

This should be fixed in 2.0.0 update.

>Also: I'd like for it to be able to ignore (for example) controllers:

This is a specific request and not everyone will like to enable it. I'll try add some configuration option to toggle this. Thanks for the suggestion.

>...and testmethods from unittests:

This should be fixed in 2.0.0 update.
